### ISSUE TITLE
Is it possible to use Retry in the feign client 

### ISSUE THREAD

**Issue by PriyankPurwar** [resilience4j/resilience4j#765]
Resilience4j version:1.1.0

Java version: 1.8

Problem description: I am looking for a way to use the Retry while  construct the Feign Client using Resilience4jFeign.builder

```
CircuitBreaker circuitBreaker = circuitBreakerRegistry.circuitBreaker("accountsClient", "accountsClient");
FeignDecorators decorators = FeignDecorators.builder()
                                                        .withCircuitBreaker(circuitBreaker)
                                                        .withFallback(accountClientFallback, CallNotPermittedException.class)
                                                        .build();
AccountClient accountResilientClient = Resilience4jFeign.builder(decorators)
                                                                    .encoder(new FormEncoder(new JacksonEncoder(objectMapper)))
                                                                    .decoder(new JacksonDecoder())
                                                                    .decode404()
                                                                    .contract(new SpringMvcContract())
                                                                    .target(AccountClient.class, baseUrl + "/" + accountsPath);
```

How I can add the Retry while building the FiegnClient

**Comment by Romeh** [comment]
@PriyankPurwar  we will add the support for it in the next release .

**Comment by PriyankPurwar** [comment]
@Romeh Thanks. Can you please share any time line you have for that

**Comment by RobWin** [comment]
Merged

**Comment by PriyankPurwar** [comment]
Hello RobWin, Are these changes available in 1.2.0 resilience4j-feign? I don't see them in this release yet

**Comment by RobWin** [comment]
No, Milestone is 1.2.1
